microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -703,8 +771,25 @@ See also:
 *  [Throttle the CPU while recording](../evaluate-performance/reference.md#throttle-the-cpu-while-recording) in _Performance features reference_.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 116.
-*  This checkbox is present in Microsoft Edge Stable 114.
+*  This checkbox is present in Microsoft Edge Canary 117.
+*  This checkbox is present in Microsoft Edge Stable 116.
+
+
+<!-- ====================================================================== -->
+## Explain Console errors with AI
+
+_new as of Microsoft Edge 118_
+
+Explains Console errors by using AI.
+<!-- todo: need text -->
```

Comment:
@captainbrosset @zoherghadyali - need real text for Experiments checkbox "Explain Console errors with AI"

Reply:
removed